streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -66,17 +67,28 @@
  *
  * StreamsConfig streamsConfig = new StreamsConfig(streamsProperties);
  * }</pre>
- * Kafka Streams required to set at least properties {@link #APPLICATION_ID_CONFIG "application.id"} and
- * {@link #BOOTSTRAP_SERVERS_CONFIG "bootstrap.servers"}.
- * Furthermore, it is not allowed to enable {@link ConsumerConfig#ENABLE_AUTO_COMMIT_CONFIG "enable.auto.commit"} that
- * is disabled by Kafka Streams by default.
+ * 
+ * Kafka Streams require at least the following properties to be set:<br>
+ * <ul>
+ *  <li>{@link #APPLICATION_ID_CONFIG "application.id"}</li>
+ *  <li>{@link #BOOTSTRAP_SERVERS_CONFIG "bootstrap.servers"}</li>
+ * </ul>
+ * 
+ * <br>Kafka Streams do not allow the following properties to be overwritten (default values
+ * set for these properties by Kafka Streams will be used instead):<br>
+ * <ul>
+ *  <li>{@link ConsumerConfig#ENABLE_AUTO_COMMIT_CONFIG "enable.auto.commit"} &nbsp-&nbsp disabled by default</li>
```

Comment:
Nit: I think we can improve the wording and simplify the markup (why do we need both `&nbsp`?). What about:

`<li>{@link ConsumerConfig#ENABLE_AUTO_COMMIT_CONFIG "enable.auto.commit"} (disabled)</li>`